### DISCUSSION TITLE
Profit Limits

### DISCUSSION THREAD

**Discussion by nealmick** [kyleskom/NBA-Machine-Learning-Sports-Betting#161] [Ideas]
Thought I would start this discussion, see if anyone has experienced a limited or banned account due to profiting (without arbitrage).

https://abcnews.go.com/Sports/won-sportsbooks-banning-smart-money/story?id=57307967

Any ideas on how to prevent bans or appear to be winning less in long run?  
How much of a problem really is this, it is reported to be growing, is this true?
Are there really bookmakers less likely to limit, or who advertise as accepting to "sharp bettors and dangerous play"?
There are many sports-books so its not the end of the world if you get banned from one, but many are owned by the same parent companies, are they sharing a black-list of people they wont allow to play?

Personally I have never been limited or banned, but I have very little experience actually betting on games.  Would love any insight or feedback.

**Comment by nealmick** [comment]
https://www.youtube.com/watch?v=LeQ71jF1Qzc&ab_channel=42Analytics

**Reply by nealmick** [reply]
Timestamps
https://youtu.be/LeQ71jF1Qzc?t=1539
https://youtu.be/LeQ71jF1Qzc?t=2417

**Comment by Gioant33** [comment]
Hi Mick, haven’t watched the entire video you linked but what I’ve found is that I’ve been getting limited online because my bets are beating the closing line. So when my CLV is very high I notice my limits coming down.

The problem is the only way to not get limited when you’re beating closing lines is by taking bets with negative EV which none of us want to do.

The only way to increase longevity is to use as many books as possible. This seems to keep heat off instead of betting in the same place constantly.

**Reply by nealmick** [reply]
Makes sense, distributing bets over different books or even decoy type bets once in a while seems right.  Ive heard PinnacleSports does not limit, but isn't available in US, any experience with them?

**Reply by Gioant33** [reply]
Yeah technically they have the best lines bc they take any bet so their line (depending on the sport and market) will be the best line because they will take all the sharp action.

Unfortunately I’m in the US so can’t use pinnacle but they usually have the strongest lines which are tough to beat

**Reply by Gioant33** [reply]
Also if anyone has any ideas for someone who can’t code (lol) I’m trying to piece together something that can scrape odds directly from sports books (legal in us) create a market average, and point out the best and worst lines for o/u, ML, and spreads. Also sorry for hijacking this thread

**Reply by nealmick** [reply]
I could probably build something to compare different sports books.  I have a deal worked out with the owner of this API https://rapidapi.com/tank01/api/tank01-fantasy-stats/  gives me 2000 request/day.  They also have injury data, and bunch of cool stuff like player ESPN images.  I think this would be a great addition to my website as-well.  https://nbadata.cloud/  
Also if kyle reads this i was thinking of hosting a public version of your project on my server, could do some cool integration with some of my data as-well like team logos!

**Reply by Gioant33** [reply]
Wow that’s really interesting I could spend all night on your website lol. I think if you could figure out a way to compare your predictions to all legal sports books with the option to choose the best odds that would really improve ROI.

Then it’s just a matter of being able to get bets down at the books with the best odds. Great work on the site too

**Reply by nealmick** [reply]
Thanks a bunch, Im gona start working on this tomorrow.  Currently I only track Fandual And Draftkings point spreads, but the api has data for MGM, Caesars, and a few others.  Not sure yet how it will be displayed, Probably make another table under the injury data.  Really great idea though, I thought about it a while back but never got around to building!

**Reply by nealmick** [reply]
Okay got the spread table about half built.  It's displaying all 8 books listed on the api, with ml spread and ou.  Currently it does not point out the best line, or sort in any way.  Im not exactly sure how i want to do that.  I could sort in some way based on the predicted scores, but I think it would just be simple and easy to sort by point spread min to max.  Then you can see the lines on either end, as the best.... and could even make like an invert button to reverse the order or something.  I also want to make an update spread button, currently the values are just gathered when the game is created, and are not updated....
example game:
https://nbadata.cloud/predict/edit/4762/

**Reply by Gioant33** [reply]
Yeah that’s great work that looks really good I think that’s very useful. IMO the games you’re predicting in margin 3 (giving you the biggest edge) would most likely have a wider discrepancy between books prices. 

Even in the game you just shared there’s a full point discrepancy which if you can get money down at those books is making your predictions even better.

good work on that! I also sent you an email not sure if you received that tho

**Reply by nealmick** [reply]
Yep, the margin is calculated using FanDuel's point spread, and often times there is a more profitable line with another book.  Cherry picking the line can improve expected value.  Also from what i've seen often times the line's move faster and are more volatile during early game when a team might get a lead.  Most likely value in catching those swings. 
I don't think that email got to me, my main email is nealmick99@gmail.com

**Reply by Gioant33** [reply]
Great I completely agree! And that sounds good I resent the email!
